.github/workflows/ci-promote-test.yaml

```diff
@@ -0,0 +1,55 @@
+name: OpenShift Build and Deploy to Dev
+
+on:
+  push:
+    tags:
+      - test
+
+env:
+  CLUSTER: https://api.gold.devops.gov.bc.ca:6443
+  AUTH_TOKEN: ${{ secrets.OC_AUTH_TOKEN }}
+  TOOLS_TOKEN: ${{ secrets.OC_TOOLS_TOKEN }}
```

Comment:
Do we use tools token for each deployment/promotion?

Reply:
I need to consider this, but let's keep this for now